bps/templates/bps_darkBootstrap.yaml

```diff
@@ -0,0 +1,16 @@
+pipelineYaml: "${CP_PIPE_DIR}/pipelines/${INSTRUMENT}/cpDarkBootstrap.yaml"
+
+project: "${TICKET}"
+campaign: "${TICKET}"
+
+darkBootstrapCollection: "${USER_CALIB_PREFIX}$INSTRUMENT/calib/$TICKET/$TAG/darkBootstrapGen.$RERUN"
+biasBootstrapRun: "${USER_CALIB_PREFIX}$INSTRUMENT/calib/$TICKET/$TAG/biasBootstrapGen.$RERUN/run${BOOTSTRAP_RUN_NUMBER}"
+darkBootstrapRun: "{darkBootstrapCollection}/run${BOOTSTRAP_RUN_NUMBER}"
```

Comment:
Same question as on biasBootstrap re `$`.

Reply:
I see `inCollection` is the same, so maybe I'm just confused about variable scope.  Those defined within the yaml only need braces, but those that are inherited from the environment need `${}`?